setup.py

```diff
@@ -6,10 +6,8 @@
 # To configure, compile, install, just run this script.
 #     python setup.py install
 
-DESCRIPTION = """Pygame is a Python wrapper module for the
-SDL multimedia library. It contains python functions and classes
-that will allow you to use SDL's support for playing cdroms,
-audio and video output, and keyboard, mouse and joystick input."""
+with open('README.rst') as readme:
```

Comment:
I think this will fix it for unicode?
```
 with open('README.rst', encoding='utf-8') as readme:
```

Does our README.rst validate with the pypi rst I wonder? I noticed twine now has a checker, we could add to our CI( `twine check dist/*`). https://packaging.python.org/guides/making-a-pypi-friendly-readme/

Reply:
Thanks for the unicode reminder, I keep forget that UTF-8 is not the default on all platforms.  I'll check for PyPI compatibility as I address this.